frontend/src/pages/CreateNewInquiry.js

```diff
@@ -22,9 +24,27 @@ export default function CreateNewInquiry(){
     }
 
     return(
-        <>
+        <PageLayoutContainer>
             <BurgerMenu/>
-            <FormReactHookCreateNewInquiry onAdd={addNewInquiry}/>
-        </>
+            <ItemContainer>
+                <FormReactHookCreateNewInquiry onAdd={addNewInquiry}/>
+            </ItemContainer>
+            {/*<FormReactHookCreateNewInquiry onAdd={addNewInquiry}/>*/}
+        </PageLayoutContainer>
     )
-}
\ No newline at end of file
+}
+
+const PageLayoutContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  height: 100vh;
+  overflow: hidden;
+  background-color: whitesmoke;
+`
+
+const ItemContainer = styled.div`
+  flex-grow: 0.1;
+  height: 100vh;
```

Comment:
I would expect this to cause overlay conflict since the surrounding `<div>` also has `height: 100vh;`
what are you trying to achieve here?
if you are trying to give the `<ItemContainer>` the remainder of the page you may want to consider switching to a grid for your `<PageLayoutContainer>` with `grid-template-columns: auto 1fr;`